bin/sdk/gutenberg.js

```diff
@@ -41,8 +40,9 @@ exports.config = ( { argv: { inputDir, outputDir }, getBaseConfig } ) => {
 	let presetBetaBlocks;
```

Comment:
Some of these `let` can probably be direct `const` since not all of theme are modified after declaring.